sig-autoscaling/charter.md

```diff
@@ -0,0 +1,78 @@
+# SIG Autoscaling Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+Covers development and maintenance of components for automated scaling in
```

Comment:
I think that there should be something about "integrated into kubernetes by default" or something like that.

Basically, I want it to be clear that if someone develops a random autoscaler in their own repo, it's not covered by this SIG.  I think that's pretty clearly understood by everyone involved, but this text seems to lay claim to all Kubernetes autoscaling

(and the thread below that points to an incubator project as part of the scope makes this distinction even more important to make)

Reply:
We also own officially "sponsored" projects.  The term "integrated in to Kubernetes by default" is pretty ambiguous, especially in the era of "do it out-of-tree using a CRD" (i.e. vertical pod autoscaler and cluster autoscaler).  If someone wants to get their project sponsored by a SIG, SIG Autoscaling is the right place, yet it may not be "integrated into Kubernetes by default".